Neptune.EFModels/Entities/OnlandVisualTrashAssessmentAreaExtensionMethods.cs

```diff
@@ -34,8 +34,8 @@ public static OnlandVisualTrashAssessmentAreaDetailDto AsDetailDto(this OnlandVi
             OnlandVisualTrashAssessmentProgressScoreName = onlandVisualTrashAssessmentArea.OnlandVisualTrashAssessmentProgressScore?.OnlandVisualTrashAssessmentScoreDisplayName,
             NumberOfAssessmentsCompleted = onlandVisualTrashAssessmentArea.OnlandVisualTrashAssessments.Count,
             LastAssessmentDate = onlandVisualTrashAssessmentArea.OnlandVisualTrashAssessments.Select(x => x.CompletedDate).Max(),
-            OnlandVisualTrashAssessments = onlandVisualTrashAssessmentArea.OnlandVisualTrashAssessments.Select(x => x.AsGridDto()).ToList(),
             BoundingBox = new BoundingBoxDto(onlandVisualTrashAssessmentArea.OnlandVisualTrashAssessmentAreaGeometry4326),
+            Geometry = onlandVisualTrashAssessmentArea.GetGeometry4326GeoJson()

```

Comment:
I would call Geometry as GeometryAsGeoJson or something like that; I assumed the Geometry property was a Geometry when I first saw it and didn't realize it was supposed to be geojson